apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -171,6 +171,25 @@ ConversationView.prototype = {
     );
   },
 
+  enterEditMode: function() {
+    this.showOptions();
+    this.menuAccessors.selectAppMenuOption('Select Messages');
+  },
+
+  exitEditMode: function() {
+    this.back();
+  },
+
+  isInEditMode: function() {
```

Comment:
Let's move this to ```test``` itself and we need to make sure that _all_ messages are in edit mode, not just one :) Something like:
```
conversationView.messages.forEach(function(message) {
  assert.isTrue(message.isInEditMode);
});
```